### PR TITLE
Update Geometry examples for explicit globals & make more Pythonic

### PR DESCRIPTION
**Pull request by tildebyte**

### CODE REVIEW

File: mode/examples/Topics/Geometry/Icosahedra/Icosahedra.pyde [hunk 1]
```diff
@@ -23,13 +19,12 @@ icoY3Rot = PI / 200
 
 
 def setup():
+    global halfWidth, halfHeight, ico3XOffset, ico1XOffset, ico1, ico2, ico3
     size(640, 360, P3D)
-    global halfWidth, halfHeight, ico3XOffset, ico1XOffset
```
Comment (jdf) [inline mode/examples/Topics/Geometry/Icosahedra/Icosahedra.pyde]: What's wrong with global statements at point of declaration?

File: mode/examples/Topics/Geometry/ShapeTransform/ShapeTransform.pyde [hunk 2]
```diff
@@ -13,23 +13,21 @@ Down Arrow - decreases points
...
 radius = 99
 isPyramid = False
 cylinderLength = 95
 
 
 def setup():
+    global halfWidth, halfHeight
     size(640, 360, P3D)
     noStroke()
-    global halfWidth, halfHeight
```
Comment (jdf) [inline mode/examples/Topics/Geometry/ShapeTransform/ShapeTransform.pyde]: Why?

Comment (tildebyte) [inline mode/examples/Topics/Geometry/ShapeTransform/ShapeTransform.pyde]: `width`/`height` are undeclared until *after* `size()`.

File: mode/examples/Topics/Geometry/Icosahedra/Icosahedra.pyde (hunk 1 above)
Comment (tildebyte) [inline mode/examples/Topics/Geometry/Icosahedra/Icosahedra.pyde]: I meant to have them all immediately following `size()`. My feeling is that declarations of symbols should *never* be hard to find (looks at C, shudders). Having globals, in particular, declared as soon as possible in a method, and grouped all together, makes them that much easier to see.

**Comment by tildebyte** [comment]
@jdf Done

**Pull request merged**